Dynatrace/dynatrace-operator
# Use raw images instead of pre-configured latest or versioned images.

# Description

Images retrieved from tenant with the `:latest` tag come pre-configured. We don't need that as the operator is configuring the agent anyway. Now we use `:raw` instead of `:latest` and e.g. `1.271.0-raw` instead of `1.271.0`.

## How can this be tested?
Deploy the operator and a dynakube and check if oneagent pods startup alright and use a `raw` image.


## Checklist
